spec/services/service_offering/add_to_portfolio_item_spec.rb

```diff
@@ -0,0 +1,68 @@
+describe ServiceOffering::AddToPortfolioItem do
+  let(:api_instance) { double }
+  let(:service_offering_ref) { "1" }
+
+  let(:add_to_portfolio_item) { described_class.new(:service_offering_ref => service_offering_ref) }
+
+  let(:params) { HashWithIndifferentAccess.new(:name => "test name", :description => "test description") }
+
+  let(:topology_service_offering) { ServiceOfferingHelper.fully_populated_service_offering }
+  let(:top_inv) do
```

Comment:
Possibly a more descriptive name here `toplological_inventory`, helps the the `before` block read better on line 16.

Reply:
Done